src/libraries/Native/AnyOS/System.Globalization.Native/pal_collation.c

```diff
@@ -419,7 +427,7 @@ int32_t GlobalizationNative_GetSortVersion(SortHandle* pSortHandle)
 {
     UErrorCode err = U_ZERO_ERROR;
     const UCollator* pColl = GetCollatorFromSortHandle(pSortHandle, 0, &err);
-    int32_t result = 0;
+    int32_t result = 0xFFFFFFFF;
```

Comment:
```suggestion
    int32_t result = -1;
```

Reply:
Just curious, why would -1 be better?

Reply:
- `-1` is shorter, easier to read
- `0xFFFFFFFF` may be prone to generating warnings because of it is outside of int32 range. Conversion from `0xFFFFFFFF` to int32 is silent overflow.

Reply:
Ok cool. Thanks for elaborating.